src/Policies.cpp

```diff
@@ -268,6 +280,9 @@ std::vector<Policy *> PolicyManager::load(const YAML::Node &policy_yaml)
 
         // make sure policy starts before committing
         try {
+            if (input_resources_policy) {
+                input_resources_policy->start();
```

Comment:
this should only start() it if it hadn't been already (although maybe we handle that in start() already)

Reply:
input_resources_policy unique ptr will only exists if new input and therefore new input_resources_policy exists. it will be called only once